lib/matplotlib/__init__.py

```diff
@@ -118,6 +118,7 @@
 from collections import namedtuple
 from collections.abc import MutableMapping
 import contextlib
+from contextlib import ExitStack
```

Comment:
You already have `contextlib` imported, not sure this saves enough characters to require a second import.

Reply:
I tend to prefer importing classes using from foo import Class, but it's not really a hard rule so I can remove this import if you prefer.